Foxtrot/Foxtrot/Utils/HelperMethods.cs

```diff
@@ -4186,8 +4186,7 @@ public static Method GetContractMethod(Method instantiatedMethod)
             
             if (candidate != null)
             {
-                if (candidate.Contract != null &&
-                    (candidate.Contract.RequiresCount > 0 || candidate.Contract.EnsuresCount > 0))
+                if (candidate.Contract != null && candidate.Contract.HasAssertions)
```

Comment:
Actually I've extracted this code from another place, not from this one. There is another place that needs to be consistent.

There is a plenty of precondition types. `ValidationCount` is a number of method calls abbreviated with `ContractValidationAttribute`. All of them should be considered in this case.